get_all_manifests.sh

```diff
@@ -10,11 +10,11 @@ declare -A COMPONENT_MANIFESTS=(
     ["ray"]="opendatahub-io:kuberay:master:ray-operator/config:ray"
     ["kueue"]="opendatahub-io:kueue:dev:config:kueue"
     ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:main:config:data-science-pipelines-operator"
-    ["odh-dashboard"]="opendatahub-io:odh-dashboard:incubation:manifests:dashboard"
+    ["odh-dashboard"]="opendatahub-io:odh-dashboard:main:manifests:dashboard"
```

Comment:
Incubation is still our ODH Nightly branch -- this is a premature change. We are not ready to chase down all the threads and make this change. Please revert this back to `incubation`.

Reply:
ok, i will not have the change for dashboard, has been waiting for weeks and i will get trusty out now.